docs/Intro_to_docs.md

```diff
@@ -0,0 +1,27 @@
+
+* Use case: A use case refers to a use of the data sharing infrastructure for a defined purpose that has associated Information Governance agreements, such as a Data Processing Agreement (DSA) and Data Protection Impact Assessment (DPIA). Use cases are agreed between the data controllers and the data processor. They define the nature of the outputs that should be produced by the pipelines and what the outputs can be used for. Note that a use case may involve more than one pipeline. For example, the "PAN" use case defines outputs to be produced for the SSDA903, CIN Census and Annex A pipelines.
+
+## Some important foundations
+
+* Data sharing infrastructure: The pipelines have been written to be run on cloud infrastructure that can be accessed by both the data controllers, who upload input data to the infrastructure, and the data processor, who retrieves output data from the infrastructure. The pipelines assume that the infrastructure has already managed the upload of data, has separated that data into distinct areas belonging to each data controller, and has labelled those areas in such a way that the controller can be identified by the pipeline code. When the pipeline code begins, it inherits from the infrastructure's scheduler the necessary information about the location of the input file to be processed and the identity of the data controller.
+
+* Schema-based validation and processing: A key foundation for the pipelines is the schema. This describes two fundamental aspects of the processing:
+    * the structure of the input datasets, including:
+        * the fields
+        * formats of data fields
+        * values allowed in categorical fields
+        * whether field values are mandatory
+    * the instructions on how to transform data to produce the outputs, including:
+        * whether files, or fields within files, should be retained for a specific output
+        * whether fields should be degraded e.g. reducing the granularity of geographical locators
+        * how long files should be retained before being deleted
+The schema for a dataset should correspond exactly to the processing detailed in the Information Governance for the use cases relevant to that dataset. An audit of the schema should be able to map every data transformation and decision to an instruction detailed in the Information Governance.
```

Comment:
This needs an extra new line to process the way I assume you want it to (as a new paragraph after the 3 bullet points, otherwise it just becomes one really long bullet point)